docs/index.rst

```diff
@@ -61,3 +61,30 @@ Features
     how-to
     how-it-works
     changelog
+
+Known limitations
+-----------------
+pytest-xdist have some limitation that may be supported in pytest but can't be supported in pytest-xdist
+
+Order and amount of test must be constant
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Is is not possible to have test that differs in order or their amount across threads/execution.
+
+This is especially true with `pytest.mark.parametrize`.
+When parametrize is used with set or other unordered iterable-like/generator
+pytest-xdist
+
+
+Example
+::
+    import pytest
+
+    @pytest.mark.parametrize("param",{"a","b"})
+    def test_pytest_parametrize_unordered(param):
+        pass
+
+will cause issue despite pytest supports it.
+
+Truncated pytest signature:
```

Comment:
IMHO this last two lines can be removed as I don't think they clarify the problem any better  (a `set` is iterable for example).

I think instead we can add a quick example on how to fix the problem, something like:

```rst
A solution to this is to guarantee that the parametrized values have the same order, for example, call `sorted` on them:

.. code-block:: python

    import pytest

    @pytest.mark.parametrize("param", sorted({"a","b"}))
    def test_pytest_parametrize_unordered(param):
        pass
```

Reply:
will add another solution with array as  1st example
since it looks to me more tidy and requires change of only 2 characters

it is always best to have multiple options to fix things